ui/global-nav.tsx

```diff
@@ -61,7 +61,12 @@ export function GlobalNav() {
 
                 <div className="flex flex-col gap-1">
                   {category.items.map((item) => (
-                    <NavItem key={item.slug} item={item} close={close} />
+                    <Suspense
+                      key={item.slug}
+                      fallback={<NavItem item={item} close={close} />}
+                    >
+                      <DynamicNavItem item={item} close={close} />
+                    </Suspense>
```

Comment:
The goal here is to put the suspense boundary as deep in the tree as possible, so that the fallback shell for routes with dynamic params can include these elements. Since `NavItem` was using `useSelectedLayoutSegment` which suspends on accessing the fallback params, we split it into a dynamic component and a skeleton component. Same for `TabContent`.